src/web/src/components/Icon/commons/LogOut01.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+const LogOut01 = () => {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="20"
+      height="20"
```

Comment:
Geralmente a `viewbox` é 24px por 24px se n for isso, desconfia que tu pegou do lugar errado